ui/MainWindow.ui

```diff
@@ -331,6 +358,16 @@
               </widget>
              </item>
              <item>
+              <widget class="QPushButton" name="genRemediationButton">
+               <property name="toolTip">
+                <string>Generate remediations of all possible failiures that the selected profile can cover.</string>
```

Comment:
typos. again, please use consistent terms. It's a remediation role. It contains a fix for every single rule in the profile that has a fix.